api/app/Listeners/StoreApplicationSnapshot.php

```diff
@@ -47,6 +47,11 @@ public function handle(ApplicationSubmitted $event)
             'poolCandidates',
             'poolCandidates.pool',
             'poolCandidates.pool.classifications',
+            'poolCandidates.awardExperiencesCriteria.skills',
+            'poolCandidates.communityExperiencesCriteria.skills',
+            'poolCandidates.educationExperiencesCriteria.skills',
+            'poolCandidates.personalExperiencesCriteria.skills',
+            'poolCandidates.workExperiencesCriteria.skills',
```

Comment:
Do we actually need the `.skills` for each of these? For minimum requirements, I think its just the experience itself which matters.

Reply:
The resource collectors for snapshots throw an error about lazy loading skills if the eager load is not present.